docs/src/pages/guides/interoperability/StyledComponentsTheme.js

```diff
@@ -1,47 +1,36 @@
 import * as React from 'react';
-import styled, { ThemeProvider } from 'styled-components';
-import NoSsr from '@material-ui/core/NoSsr';
 import {
   createMuiTheme,
+  experimentalStyled as styled,
   ThemeProvider as MuiThemeProvider,
   darken,
 } from '@material-ui/core/styles';
-import Button from '@material-ui/core/Button';
+import Slider from '@material-ui/lab/SliderStyled';
+import Box from '@material-ui/core/Box';
 
 const customTheme = createMuiTheme({
   palette: {
     primary: {
-      main: '#6772e5',
+      main: '#20b2aa',
     },
   },
 });
 
-const StyledButton = styled(Button)`
+const CustomizedSlider = styled(Slider)`
   ${({ theme }) => `
```

Comment:
How about we apply https://github.com/mui-org/material-ui/pull/22964#discussion_r502707299 here too for the simplicity of memorizing the synthax?